pkg/scheduler/util/predicate_helper.go

```diff
@@ -107,3 +107,47 @@ func taskGroupID(task *api.TaskInfo) string {
 func NewPredicateHelper() PredicateHelper {
 	return &predicateHelper{taskPredicateErrorCache: map[string]map[string]error{}}
 }
+
+type PredicateStatus interface {
+	IsContainsUnschedulable() bool
```

Comment:
The `IsContainsUnschedulable` is inconsistent with implementation.

Reply:
It's supposed to be deleted here, done.